package.json

```diff
@@ -3,11 +3,14 @@
   "version": "1.0.0",
   "description": "Electron app build using ReactJS and Parcel web application bundler",
   "main": "src/electron.js",
+  "targets": {
+    "main": false
+  },
```

Comment:
Parcel 2 looks at `package.json#main` (even though it's intended for Electron). Adding this tells Parcel to ignore it.

https://github.com/parcel-bundler/parcel/issues/3500#issuecomment-924777899